src/api/store/store.jsx

```diff
@@ -1,4 +1,5 @@
 import { combineReducers, configureStore } from "@reduxjs/toolkit";
+import { GetLocalStorage } from "../../helpers";
```

Comment:
function name casing is wrong - getLocalStorage, setLocalStorage

Reply:
done